src/shared/components/mutationTable/column/VariantTypeColumnFormatter.tsx

```diff
@@ -0,0 +1,152 @@
+import * as React from 'react';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import { Mutation } from 'shared/api/generated/CBioPortalAPI';
+import CategoricalColumnFormatter, {
+    ICategoricalColumn,
+} from './CategoricalColumnFormatter';
+import styles from './variantType.module.scss';
+
+/**
+ * Variant Type Column Formatter.
+ *
+ * This class follows the official GDC MAF Variant Type List:
+ * https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/
+ *
+ * Possible variant type values include:
+ * SNP, DNP, TNP, ONP, INS, DEL, or Consolidated.
+ */
+export default class VariantTypeColumnFormatter {
+    public static get MAIN_VARIANT_TYPE_MAP(): {
+        [key: string]: ICategoricalColumn;
+    } {
+        return {
+            SNP: {
+                toolTip: 'Single Nucleotide Polymorphism',
+                className: 'variant-type-snp',

```

Comment:
no big deal but you could probably reference the style on styles import itself right here (and not do the lookup later).    then you would get autocomplete on these names in your IDE, assuming it can handle css modules (WebStorm can).       I don't think dashes in class names would work anymore so you'd have to change those.     
so, i.e.,  { ..., className: styles.variantTypeSNP } 
the goal being to avoid repeated and unverifiable strings.

Reply:
thanks @alisman.  I am not familiar with this.  Can you point me to some other existing code where I can see how this works?

Reply:
ah, never mind.  I see what you mean now.  Clever idea!  Will definitely change.

Reply:
this is now done.